pkgs/by-name/tr/transmission_4-macos-bin/package.nix

```
@@ -0,0 +1,54 @@
+{
+  stdenv,
+  lib,
```

This allows to re-use transmission's meta info (see below)
```suggestion
  lib,
  transmission,
```